src/mantine-core/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import {
+  Box,
+  BoxProps,
+  StylesApiProps,
+  factory,
+  ElementProps,
+  useProps,
+  useStyles,
+  createVarsResolver,
+  Factory,
+  MantineRadius,
+  MantineColor,
+  getRadius,
+  getThemeColor,
+} from '../../core';
+import classes from './Notification.module.css';
+import { Loader } from '../Loader';
+import { Text } from '../Text';
+import { CloseButton } from '../CloseButton';
+
+export type NotificationStylesNames =
+  | 'root'
+  | 'icon'
+  | 'loader'
+  | 'body'
+  | 'title'
+  | 'description'
+  | 'closeButton';
+export type NotificationVariant = string;
+export type NotificationCssVariables = {
+  root: '--notification-radius' | '--notification-color';
+};
+
+export interface NotificationProps
+  extends BoxProps,
+    StylesApiProps<NotificationFactory>,
+    ElementProps<'div', 'title'> {
+  variant?: string;
+
+  /** Called when close button is clicked */
+  onClose?(): void;
+
+  /** Notification line or icon color */
+  color?: MantineColor;
+
+  /** Key of theme.radius or any valid CSS value to set border-radius, theme.defaultRadius by default */
```

Comment:
theme.radius, border-radius and theme.defaultRadius must be wrapped with backticks to be highlighted as code in docs